.env.py

```diff
@@ -0,0 +1,8 @@
+DATABASE_HOSTNAME = localhost
```

Comment:
usually you don't publish .env file with credentials as it is sensitive data. If you like, you can publish .env-example, but I suggest you add .env to gitignore